### DISCUSSION TITLE
Feedback for Currency Calculator

### DISCUSSION THREAD

**Discussion by 001138477** [001138477/Tafe2022S213#11] [General]
Hey Physeth, how does my Currency Calculator look, have you got any suggestions for what I should change or do I push it now?
![image](https://user-images.githubusercontent.com/112138075/191442071-872c24d9-0be8-40a7-a3fb-642b7f0feb06.png)

**Comment by PhysethLeav** [comment]
Eveyrthing looks good, yes push it
Thanks
